docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. [Flatcar Linux Update Operator
+(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) can keep the operating system
+updated every time there is a new Flatcar release. FLUO is a supported Lokomotive component. This
+guide will show how to install it and disable its auto-update feature for specific nodes.
+
+## Prerequisites
+
+- A Kubernetes cluster accessible via `kubectl`.
+
+## Steps
+
+### Step 1: Disable auto-update on nodes
```

Comment:
Hm, this is misleading. Looking at the header I thought we talk about disabling `locksmithd`. 

Perhaps we should add an optional step for it for non Lokomotive clusters?

Then, I'd change this title and perhaps mention FLUO here and maybe say something like "on sensitive nodes"? Because just saying `on nodes` seems to me like it referes to ALL nodes, which is incorrect.

Reply:
I don't think this document is a generic for anybody using FLUO anywhere. This is written with Lokomotive user's priority in mind. For a generic document the right place for it is in FLUO repo.

Reply:
Well, this is for FLUO **component** IMO, which is part of Lokomotive. Also, Lokomotive itself disables `locksmithd`, but components can be used on all kinds of clusters.

Reply:
Yep I agree that FLUO can be used on all kinds of clusters. But we can't guarantee how a certain Lokomotive component will behave on a third-party cluster because there are assumptions made about the environment (in this case Lokomotive cluster). 

For example, we cannot guarantee component security on a cluster that does not enable PSP. Similarly we cannot guarantee network security to the machines because the third party cluster might not have something similar to calico globalnetworkpolicies the way we ship them.

Given these assumptions we can omit unnecessary details a Lokomotive user has to know. In this case they care about updating the OS that powers Lokomotive and they want a way to do it. They don't wanna know how what `locksmithd` is since we take care of it for them.

Reply:
Well, I think treating components as cluster-agnostic, even when the compatibility is not guaranteed is something valuable. What do you think @iaguis?